rmc/resources/reference_data.py

```diff
@@ -37,10 +41,67 @@
 Table containing transcript start and stop positions displayed in the browser.
 """
 
+transcript_ref = TableResource(
+    path=f"{REF_DATA_PREFIX}/ht/canonical_transcripts_genes_coordinates.ht"
+)
+"""
+Table containing canonical transcripts with key reference info:
+gene name from gnomAD annotations, gene name from GENCODE v19,
+Ensembl gene ID in GENCODE v19, and overall and CDS start and end
```

Comment:
```suggestion
Ensembl gene ID in GENCODE v19 and overall CDS start and end
```

Reply:
This actually means that both the transcript overall start and end coordinates as well as the CDS start and end coordinates are annotated